osu.Game.Rulesets.Osu/Objects/Drawables/Connections/FollowPoint.cs

```diff
@@ -43,5 +44,12 @@ public FollowPoint()
                 }
             }, confineMode: ConfineMode.NoScaling);
         }
+
+        public void Fail()
```

Comment:
is this not going to fail on rewind?

Reply:
Fixed by switching between moving transforms sequence (normal one) and failing transform sequences based on `HasFailed` value. https://github.com/ppy/osu/blob/8dfcb15da8add5c94b30c3034d69ddf518ed418c/osu.Game.Rulesets.Osu/Objects/Drawables/Connections/FollowPointRenderer.cs#L103-L121